openfe/tests/protocols/openmm/plain_md/test_simulation_slow.py

```diff
@@ -0,0 +1,144 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+import pathlib
+import pytest
+from openff.units import unit
+from gufe.protocols import execute_DAG
+from openfe.protocols import openmm_md
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize('platform', ['CPU', 'CUDA'])
+def test_vacuum_sim(
+    benzene_vacuum_system,
+    platform,
+    available_platforms,
+    tmpdir
+):
+    if platform not in available_platforms:
```

Comment:
This is nice, eventually we will want to set things up so we can "force" the GPU tests so we can catch issues like the GPU drivers not working BUT for fast tests, I think this is a great idea

Reply:
Not me unfortunately, there's a good chance this was you from a previous test :P